src/java.base/share/classes/java/lang/Comparable.java

```diff
@@ -100,7 +105,7 @@
      * than, equal to, or greater than the specified object.
      *
      * <p>The implementor must ensure
-     * {@code sgn(x.compareTo(y)) == -sgn(y.compareTo(x))}
+     * {@link Integer#signum signum}{@code (x.compareTo(y)) == -signum(y.compareTo(x))}
      * for all {@code x} and {@code y}.  (This
      * implies that {@code x.compareTo(y)} must throw an exception iff
```

Comment:
I'd suggest replacing "iff" with "if and only if" because it looks like a typo, and writing out the long form emphasizes the bidirectional nature of the implication.

Reply:
Sure; most of the terminology in the JDK docs aren't very "math-y".